.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,70 @@
+name: Installation or build bug report
```

Comment:
```suggestion
name: Bug report
```

We don't 'install' anything in the sense as esp-idf must be installed. So simple 'Bug report' should suffice :)